sdk/python/pulumi_kubernetes/authentication/v1beta1/TokenReview.py

```diff
@@ -48,6 +53,12 @@ def __init__(self, resource_name, opts=None, metadata=None, spec=None, status=No
             __props__,
             opts)
 
+    @staticmethod
+    def get(name: str, id: Input[str], opts: Optional[ResourceOptions] = None):
+        opts_ = copy(opts) if opts is not None else ResourceOptions()
+        opts_.id = id
+        return TokenReview(name, opts_)
```

Comment:
so i don't see how these work since they dont' have the 'output' props to pass along.

Reply:
I may be completely misunderstanding how to do this, but it seemed like it worked when I tested it. Am I doing something obviously wrong here?

Reply:
So, the highest order bit is simply that the python impl should match the JS impl.   Looking at the JS impl, it looks like this is the case :)   So we can go ahead with this.